lib/course_planner/Mailer/main.ex

```diff
@@ -0,0 +1,6 @@
+defmodule CoursePlanner.Mailer.Main do
```

Comment:
hmm, i prefer changing the folder structure, but rather needs time. so i'll think about it